utils.py

```diff
@@ -30,6 +30,29 @@ def die(string):
     sys.exit(1)
 
 
+def find_first_file(relative_root, possible_files):
+    """
+    Attempts to find the first option available in the list of files relative
+    to a specified root folder.
+
+    Parameters:
+        relative_root (Path): A Path object containing the folder to search for
+                              files within.
+        possible_files (list): A list of Paths that may be within the relative
+                               root folder. They will be automatically appended
+                               to relative_root.
+    Returns:
+        The full path to the first file found in the list. If none could be
+        found, an Exception is raised.
+    """
+    for possible_file in possible_files:
+        if (full_path := relative_root.joinpath(possible_file)).exists():
```

Comment:
`:=` neat! When did python get that?

Reply:
3.8! https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions